test/OpenTelemetry.Tests.Stress/Skeleton.cs

```diff
@@ -169,4 +208,16 @@ private static ulong GetCpuCycles()
 
         return cycles;
     }
+
+    protected partial class StressTestOptions
+    {
+        [Option('c', "concurrency", HelpText = "The concurrency (number of executing threads) for the stress test. Default value: Environment.ProcessorCount.", Required = false)]
```

Comment:
Is "concurrency" really the same as "number of executing threads"? (most likely no)

Reply:
I updated to use "maximum degree of parallelism" which is the terminology the Parallel API is using ex https://learn.microsoft.com/dotnet/api/system.threading.tasks.paralleloptions.maxdegreeofparallelism.